src/man/sssd.conf.5.xml

```diff
@@ -560,6 +560,15 @@
                                 in a random order for each parent domain.
                             </para>
                             <para>
+                                Note that when this option is set the output
+                                format of all commands is always
+                                fully-qualified, even when using short names.
+                                In case the administrator wants the output not
+                                fully-qualified, the full_name_format option
+                                can be used as shown below:
+                                <quote>full_name_format=%1$s</quote>
```

Comment:
I really think there needs to be a big warning saying that setting the full_name_format can only be done if you are absolutely sure there are no overlapping usernames between domains. Maybe also a short explanation why, although I wonder if that would be digestable for admins.

What about something like "note that during login, the login applications often canonicalize the username by calling getpwnam(3) which, if a shortname is returned for a qualified input (while trying to reach a user which exists in multiple domains) might re-route the login attempt into the domain which uses shortnames". We might also add a link to ticket #3403.